roles/gaia/tasks/main.yml

```diff
@@ -227,5 +227,11 @@
     name: nginx-ssl
   when: gaiad_use_ssl_proxy | default(false) | bool
 
+
+- name: Rebooting the machine
```

Comment:
Thoughts:
- What exactly does the reboot help with? Is it for the SSL stuff?
- Do any other roles need this?
- Should it maybe go into `common`?
- What about placing a file on the FS that signifies this machine has been rebooted so that we can auto-detect if it's been rebooted before?
- Any downsides to rebooting every time? (takes longer?)

Reply:
It helps to make sure that updates are applied like for the kernel.
It also helps to make sure services start on boot.
Rebooting each time will you will have to wait till the machine is booted before doing work on it again.